### PR TITLE
Update html report with scrollable table and table Download

### PR DESCRIPTION
**Pull request by senthil10**
With this new template, it is enough to send just the `html` file to the user as the *tab seperated* `txt` files of table content can be downloaded from the `html`

Also, the new file `html_pandoc_scrollable.html` is just an updated copy *(only lines 86-128 and 255-397 are newly added)* of `html_pandoc.html`. Reason for preserving the old template is to be able to generate report in old format, for time being.

### CODE REVIEW

**Comment by alneberg** [comment]
Sorry to be a grumpy old man. 👴 But as a previous user I would prefer to have the tab separated table accessible directly on the command line since it is much faster than first viewing the html which usually has to be downloaded to the laptop...

**Comment by ewels** [comment]
> But as a previous user I would prefer to have the tab separated table accessible directly on the command line since it is much faster than first viewing the html which usually has to be downloaded to the laptop...

This is a good point. I guess we can keep the new template and the separate CSV file though. We don't need the old template to generate that.

**Comment by senthil10** [comment]
Thank you @alneberg and @ewels for the comments :)

@alneberg Good point 👍 just to clarify if I understand your point right, you prefer to not open the `html` at all and just access the `txt` files ? I assume you wanted this in cluster ? In any case I will say what I had in my mind.

This new change will only affect the generated `html` file, which means it will still generate the `txt` files. The reports will be generated in `irma` and then copied to delivery directory along with the data. So in the delivery directory, the `reports` folder will still consist of `html` (new format) and `txt` files. We bioinformaticians, usually attach a copy of all this files in the delivery email. The main purpose of this is to reduce the number of files attached to the delivery email and also I think it is good in a way, as it would indirectly enforce the user to look at the report atleast once (if they are looking uo the report in their laptop). We have got questions from users for which they could find answers in the report 😜 

@ewels Yes, I too believe `99.99%` there wont be any need at all for the old template. But the "paranoid" me felt maybe I keep it as fail safe until this had some run in `irma` for a while 😉 As it is really a tiny job I thought I could remove it when I do next next deployment in `irma`

**Comment by alneberg** [comment]
Yes you're correct @senthil10 and I would be very happy with your suggested solution. 👍 Sorry for misunderstanding. :)

**Comment by senthil10** [comment]
>Yes you're correct @senthil10 and I would be very happy with your suggested solution. 👍 Sorry for misunderstanding. :)

Not a problem @alneberg , it is always good to discuss the PRs 😄 👍 

And @ewels , on second thought I think i will remove the old version. Will make a commit and let you know.

**Comment by senthil10** [comment]
Removed the old template @ewels 👍

File: data/pandoc_templates/html_pandoc.html [hunk 1]
```diff
@@ -83,6 +83,50 @@
...
+    width: 70px;
+    height: 14px;
+    color:#626466;
+    font-size: 10px;
+    font-weight: 550;
+    text-align: center;
+    padding: 2px 0px 2px 2px;
+    line-height: 15px;
+    border-radius: 3px;
+    border-width: 0.5px;
```
Comment (ewels) [inline data/pandoc_templates/html_pandoc.html]: Minor, but you can just use `border` to do all of this in one go:

```
border: 0.5px solid #999999;
```

**Pull request merged**